presto-main/src/main/java/com/facebook/presto/sql/planner/plan/StatisticAggregations.java

```diff
@@ -25,31 +25,51 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 
+import javax.annotation.concurrent.Immutable;
+
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static java.util.Objects.requireNonNull;
 
 public class StatisticAggregations
 {
+    // Added to preserve the order for serializing and deserializing
+    // map in c++ worker doesn't represent insertion order
+    private final List<VariableAggregation> aggregationList;
```

Comment:
> VariableAggregation

This name it confusing. It reads as "variable aggregation", not as a "{variable, aggregation} pair". I wonder if there is a better name.